avm/res/network/virtual-network-gateway/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -71,20 +60,6 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
     gatewayType: 'Vpn'
     vNetResourceId: nestedDependencies.outputs.vnetResourceId
     activeActive: true
-    diagnosticSettings: [
```

Comment:
I'm sorry to say, but I'm fairly certain diagnosticSettings are usually required for WAF-alignment. :D